app/controllers/admin_applications_controller.rb

```diff
@@ -0,0 +1,11 @@
+class AdminApplicationsController < ApplicationController
```

Comment:
AdminApplications works as a name but it makes it seem there's an Admin table and this is a joins controller between Admin and Application tables. Admin may be more clear here

Reply:
This is a unique functionality, and though not a joins function, it has it's own unique show view, different from ApplicationsController.

Reply:
I would just rename it AdminController. ATP it comes down to preference like most things in code